src/lib/runtime_config/runtime_config.ml

```diff
@@ -661,6 +755,21 @@ module Proof_keys = struct
     }
   [@@deriving bin_io_unversioned]
 
+  let make ?level ?sub_window_per_window ?ledger_depth ?work_delay
+      ?block_window_duration_ms ?transaction_capacity ?coinbase_amount
+      ?supercharged_coinbase_factor ?account_creation_fee ?fork () =
+    { level
+    ; sub_windows_per_window = sub_window_per_window
```

Comment:
Why this renaming? The original is more accurate

Reply:
Absolutely no reason. Removed in 074f6a71cf.